OutRain-API/README.md

```diff
@@ -1,4 +1,4 @@
-# Core Services Exercise
+# OutRain Exercise
```

Comment:
open question - do we need this exercise at this point? I think consul-api does a good job to cover this as well